ogusa/constants.py

```diff
@@ -101,8 +101,11 @@
                 'I': 'Investment-Output Ratio ($I_{t}/Y_t$)',
                 'total_tax_revenue': 'Tax Revenue-to-GDP ($REV_{t}/Y_t$)'}
 
-GROUP_LABELS = {0: '0-25%', 1: '25-50%', 2: '50-70%', 3: '70-80%',
-                4: '80-90%', 5: '90-99%', 6: 'Top 1%'}
+GROUP_LABELS = {7: {0: '0-25%', 1: '25-50%', 2: '50-70%', 3: '70-80%',
```

Comment:
@jdebacker . We have a `7` specification here and a `10` specification here. Should we also have the `9` specification in this dictionary?

Reply:
Yes - just added.